File: governance/tags/tools/fixtures/code-tags/sample.py
def create_player(payload):
    """
    domainspec:
      concept:
        id: player-management.CreatePlayer
        type: Operation
        concern: biz
        spec_ref:
          path: docs/features/payments/SPEC.md
          line: 11
          section: Concept Registry
      edges:
        - edge: produces
          to: player-management.PlayerCreated
    """
    return {"ok": True, "payload": payload}

File: governance/tags/tools/fixtures/code-tags/sample.js
/**
 * domainspec:
 *   concept:
 *     id: payment.ProcessPayment
 *     type: Operation
 *     concern: biz
 *     spec_ref:
 *       path: docs/features/payments/SPEC.md
 *       line: 8
 *       section: Concept Registry
 *   edges:
 *     - edge: produces
 *       to: payment.PaymentInitiated
 */
export async function processPayment(payload) {
  return { ok: true, payload };
}
